indexer-app/src/cmr/indexer/data/concepts/generic.clj

```diff
@@ -74,7 +75,7 @@
         generic-associations (esearch/parse-non-tombstone-associations
                               context
                               (meta-db/get-generic-associations-for-concept context concept))
-        gen-name (util/safe-lowercase (get-in parsed-concept [:MetadataSpecification :Name]))
+        gen-name (csk/->kebab-case (get-in parsed-concept [:MetadataSpecification :Name] ""))
```

Comment:
safe-lowercase is to guard the nil case. (csk/->kebab-case nil) will throw exception.

Reply:
Yes, that is what we discussed this morning at standup, and the suggestion proposed (which is shown here) was to add a default case to csk/->kebab-case, which is what the empty string is. In other words, `(csk/->kebab-case nil)` throws exception, but `(csk/->kebab-case nil "")` does not.